source/Projectile.js

```diff
@@ -51,12 +67,31 @@ export default class Projectile extends Pixi.Sprite {
               this.destroy()
           }
       }
+
+      if(this.projectileType == "mine" || this.projectileType == "superMine") {
+          this.rotation += 1
+      }
+
+      if(this.projectileType == "superMine") {
+          this.scale.x = 3
+      }
+
   }
 
 
   onCollision(collidedWith) {
-      if(this.piercing != true)
+      if(this.projectileType == "mine") {
+          this.onHit.explodeEnemy(this)
+          this.destroy()
+      }
+      if(this.projectileType == "superMine") {
+          this.onHit.explodeEnemy(this)
+          this.destroy()
+      }
```

Comment:
Actually, I was right in what I said in Slack. You're only calling `this.destroy()`. You still need to call `collidedWith.destroy()` to remove the trashbot, as `this.destroy()` only removes the projectile.